content/news/XXXX-XX-XX-mixxx-2-3-4-released.md

```diff
@@ -0,0 +1,32 @@
+title: Mixxx 2.3.4 Released
+authors: Nikolaus Einhauser
+status: draft
+tags: 2.3, 2.3.4, release announcement
+comments: yes
+
+
+You know the drill, another 2.3 bugfix release. [Mixxx
+2.3.4](https://github.com/mixxxdj/mixxx/releases/tag/2.3.3) to be exact.
+
+Let's take a look at the changes:  
+macOS received a bunch of love, especially in regards to possible UI performance
+problems and translation issues. We also enabled support for macOS 13 (Ventura)
+on Intel-CPUs, while there is still some work in progress to support Apple
+silicon from Mixxx 2.4.0.  
+In addition, our Tango and LateNight skins received a lot of minor tweaks and
+visual fixes.  
+Issues occurring during metadata import from 3rd party software were also fixed.
+This includes issues when fetching data from MusicBrainz as well as importing
+metadata from Rekordbox Databases or Music Files containing Serato metadata.
```

Comment:
```suggestion
metadata from Rekordbox databases or music files containing Serato metadata.
```
or was there a reason for capitals?

Reply:
Good catch, no there really wasn't, I just didn't spot them while proofreading.

Reply:
Also applies to Changelog below, but that could also be an exception because it's a title? don't care much, nit-picking..

Reply:
You're right, also fixed "GitHub".